shared/chat/blocking/block-modal/index.tsx

```diff
@@ -388,7 +393,19 @@ class BlockModal extends React.PureComponent<Props, State> {
 
 export default BlockModal
 
-const getListHeightStyle = (numOthers: number) => ({height: numOthers >= 4 ? 380 : numOthers * 120})
+const getListHeightStyle = (numOthers: number, expanded: boolean) => ({
+  height:
+    120 +
+    (numOthers >= 1
+      ? // "Also block others" is 41px, every row is 2 * 40px rows + a 1px divider.
+        // We cap the count at 4 but even that is greater than the max modal height in Keybase.
+        41 + (numOthers >= 4 ? 4 : numOthers) * 81
```

Comment:
The problem I see here is that if it's cut off at exactly 4 people, then it won't be apparent that it's scrollable and there are more people not showing. That's why I used 380 before instead of 360.